charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -13,8 +13,20 @@ spec:
     updateMode: {{ .Values.vpa.updatePolicy.updateMode | quote }}
   resourcePolicy:
     containerPolicies:
-    - containerName: '*'
+    - containerName: machine-controller-manager-provider-vsphere
       minAllowed:
         cpu: 30m
         memory: 40M
+      maxAllowed:
+        cpu: 2
+        memory: 5G
+      controlledValues: RequestsOnly
```

Comment:
I don't think we should be switching on `controlledValues: RequestsOnly` here. We didn't adjust the Limits to the values we've actually seen in production, so telling VPA to not scale Limits is pretty dangerous here.